admin/make_homebrew_recipe.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Create a Homebrew recipe for Flocker, using the VERSION environment variable.
+
+Invoke using e.g.:
+  $ export VERSION=0.1.2
+  $ ./make_homebrew_recipe.py > flocker-${VERSION}.rb
+
+Inspired by https://github.com/tdsmith/labmisc/blob/master/mkpydeps.
+"""
+
+from os import environ
+from json import load
+from urllib2 import urlopen
+from hashlib import sha1
+from tl.eggdeps.graph import Graph
+
+version = environ.get("VERSION")
+if version is None:
+    raise Exception("Set the VERSION environment variable.")
+
+url = ("http://storage.googleapis.com/archive.clusterhq.com/"
+       "downloads/flocker/Flocker-{version}.tar.gz").format(version=version)
```

Comment:
There should at least be some validation and error reporting when the supplied sdist URL / file is invalid. I got 
```
$ VERSION=0.3.3 ./admin/make_homebrew_recipe.py 
Traceback (most recent call last):
  File "./admin/make_homebrew_recipe.py", line 27, in <module>
    tarball = urlopen(url)
  File "/usr/lib64/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib64/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib64/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib64/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
urllib2.HTTPError: HTTP Error 404: Not Found

```

Which didn't help much.